lms/djangoapps/learner_dashboard/tests/test_views.py

```diff
@@ -114,18 +120,19 @@ def test_api_returns_discussions_iframe(self):
         self.assertIsInstance(response.data['discussion']['iframe'], Markup)
         self.assertIn('iframe', str(response.data['discussion']['iframe']), )
 
-    def test_program_does_not_exist(self):
+    def test_program_empty_response(self):
         """
-        Test if API returns 404 in case program does not exist
-        """
-        response = self.client.get(reverse('program_discussion', kwargs={'program_uuid': str(uuid4())}))
-        self.assertEqual(response.status_code, 404)
```

Comment:
why this test is removed?

Reply:
Because the permission has been removed so this case does not occur anymore

Reply:
got it thanks - resolving this conv.